src/ServiceFabricPubSub/ServiceFabricPubSub.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 15
-VisualStudioVersion = 15.0.26430.6
```

Comment:
Why did the Visual Studio revision number go down? Shouldn't this be 15.0.26430.6? Granted, it is a very minor change according to https://www.visualstudio.com/en-us/news/releasenotes/vs2017-relnotes.